setup.cfg

```diff
@@ -0,0 +1,2 @@
+[metadata]
+description-file = README.md
```

Comment:
is this variable used somewhere else?

Reply:
I am not 100% sure but I think it's pypi.